### PR TITLE
add note about installing docstring parser to readme

### PR DESCRIPTION
**Pull request by ym-han**
Hi Wen,

Thanks for this awesome package! I noticed that I had to install a dependency for this to work, and thought I would add a quick note to that effect.

### CODE REVIEW

**Comment by wenkokke** [comment]
Ah! I intended for it to work without that package, and only have additional functionality if it were present! Did I mess up?

**Comment by ym-han** [comment]
Ah, I basically installed it just because I'd noticed my Talon log saying that it wasn't found. But upon closer inspection, I see that you've wrapped the import in a try except (https://github.com/wenkokke/talon-cheatsheet/search?q=docstring_parser&type=code), so maybe it didn't matter after all. Sorry for the bother if that was the case!

If it's optional but still useful, perhaps it'd still be helpful to say in the readme that it might be worth installing this package for additional functionality?

Also, just as an unrelated fyi, I had to change the absolute imports to relative ones for the scripts to work on my setup. I don't know offhand why I had to do that --- might be something about my setup.

**Comment by wenkokke** [comment]
It depends on where in your user directory the code is placed.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ To recreate what I did:
 
 1. Install Talon on your computer (see [Getting Started][talon-getting-started]).
 2. Clone this repository into your Talon user directory (see [Getting Scripts](talon-getting-scripts)).
-3. Say `print help` or `print latex help`.
+3. Install a package that this cheatsheet code relies on by doing `~/.talon/bin/pip install docstring_parser` in your terminal.
```
[the remark this answers is not in the document]
Comment (ym-han) [reply README.md]: Does the commit below help?

**Review by wenkokke — APPROVED** [review]

**Pull request merged**